configs/transfer_learning.py

```diff
@@ -0,0 +1,10 @@
+from models.inception_v3 import InceptionV3
+
+# NOTE: unfreeze breakpoints and chosen model are generally codependent
+TRANSFER_LEARNING_BASE_MODEL = InceptionV3
+FINE_TUNING_UNFREEZE_BREAKPOINTS = ('mixed8', 'mixed6', 'mixed4')
```

Comment:
edit base model and breakpoints here

Reply:
Is there docs we can use to find which breakpoints for a given model?

Reply:
No don't think so unfortunately. I got these values by flipping through the layers printed by `base_model.summary()`; for InceptionV3 there are 10 mixed layers to close out each block so this just tells it to gradually unfreeze 60% of the feature extractor layers over 3 fine tuning breakpoints